src/test/java/space/obminyashka/items_exchange/service/UserServiceIntegrationTest.java

```diff
@@ -88,8 +85,17 @@ void testSelfDeleteRequest_WhenDataCorrect_Successfully() {
 
         assertEquals("ROLE_SELF_REMOVING", userWithOldPassword.getRole().getName());
         verify(userRepository).saveAndFlush(userWithOldPassword);
+
     }
+    @Test
+    void makeAccountActiveAgain_WhenDataCorrect_Successfully() {
```

Comment:
The test isn't valid at all.

1. You set the new username
2. You set "ROLE_USER"
3. You invoke the tested method
4. You validate that the user has the "ROLE_USER" but you did that in step 2.

How the tested method's result is compared?